src/main/java/frc/trigon/robot/poseestimation/t265/T265.java

```diff
@@ -0,0 +1,65 @@
+package frc.trigon.robot.poseestimation.t265;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.networktables.*;
+import org.littletonrobotics.junction.Logger;
+
+public class T265 {
+    private static T265 INSTANCE;
+    private final NetworkTable nt4Table = NetworkTableInstance.getDefault().getTable("T265");
+
+    private final IntegerSubscriber t265FPS = nt4Table.getIntegerTopic("FPS").subscribe(0);
+    private final DoubleSubscriber t265Battery = nt4Table.getDoubleTopic("BatteryLevel").subscribe(0.0f);
+    private final FloatArraySubscriber t265Position = nt4Table.getFloatArrayTopic("Position").subscribe(new float[]{0.0f, 0.0f, 0.0f});
+    private final FloatArraySubscriber t265Rotation = nt4Table.getFloatArrayTopic("Rotation").subscribe(new float[]{0.0f, 0.0f, 0.0f});
```

Comment:
no need to write t265 everytime. They're all in the T265 class it's obvious. You can write camera instead if you want.